src/utils/balloonAnimation.ts

```diff
@@ -0,0 +1,17 @@
+import styles from '../App.module.css';
+export const createBalloons = (container: HTMLDivElement | null) => {
```

Comment:
nice! while this works, there's a way to re-write this to match more of a React syntax, without the `.createElement` and `appendChild` functions. While this seems like a minor difference, it's good to consider in terms of tying a container into the component lifecycle